test/metronome_SUITE.erl

```diff
@@ -0,0 +1,72 @@
+%% Copyright (c) 2007-2016 Pivotal Software, Inc.
+%% You may use this code for any purpose.
+
+-module(metronome_SUITE).
+
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("common_test/include/ct.hrl").
+-include_lib("amqp_client/include/amqp_client.hrl").
+
+-compile(export_all).
+
+all() ->
+    [
+     {group, non_parallel_tests}
+    ].
+
+groups() ->
+    [
+     {non_parallel_tests, [], [
+                               receive_tick
+                              ]}
+    ].
+
+%% -------------------------------------------------------------------
+%% Testsuite setup/teardown.
+%% -------------------------------------------------------------------
+
+init_per_suite(Config) ->
+    rabbit_ct_helpers:log_environment(),
+    Config.
+
+end_per_suite(Config) ->
+    Config.
+
+init_per_group(_, Config) ->
+    Config1 = rabbit_ct_helpers:set_config(Config, [
+                                                    {rmq_nodename_suffix, ?MODULE}
+                                                   ]),
+    rabbit_ct_helpers:run_setup_steps(Config1,
+                                      rabbit_ct_broker_helpers:setup_steps() ++
+                                      rabbit_ct_client_helpers:setup_steps()).
+
+end_per_group(_, Config) ->
+    rabbit_ct_helpers:run_teardown_steps(Config,
+                                         rabbit_ct_client_helpers:teardown_steps() ++
+                                         rabbit_ct_broker_helpers:teardown_steps()).
+
+init_per_testcase(Testcase, Config) ->
+    rabbit_ct_helpers:testcase_started(Config, Testcase).
+
+end_per_testcase(Testcase, Config) ->
+    rabbit_ct_helpers:testcase_finished(Config, Testcase).
+
+%% -------------------------------------------------------------------
+%% Testcases.
+%% -------------------------------------------------------------------
+
+receive_tick(Config) ->
+    Channel = rabbit_ct_client_helpers:open_channel(Config),
+    #'queue.declare_ok'{queue = Q}
+        = amqp_channel:call(Channel, #'queue.declare'{exclusive = true,
+                                                      auto_delete = true}),
+    #'queue.bind_ok'{}
+        = amqp_channel:call(Channel, #'queue.bind'{queue = Q,
+                                                   exchange = <<"metronome">>,
+                                                   routing_key = <<"#">>}),
+    timer:sleep(2000),
+    case amqp_channel:call(Channel, #'basic.get'{queue = Q, no_ack = true}) of
+        {'basic.get_empty', _} -> exit(metronome_didnt_send_message);
```

Comment:
Instead of an exit signal, use either an assertion (with EUnit or not) or throw an exception. common_test logs will be more informative.